esp/services/ws_fs/ws_fsService.cpp

```diff
@@ -3444,6 +3459,9 @@ bool CFileSprayEx::onDeleteDropZoneFiles(IEspContext &context, IEspDeleteDropZon
             if (!file || !*file)
                 continue;
 
+            if (findPathSepChar(file))
```

Comment:
Can we move this parameter validation to the top, just below 
```
 if (!files.ordinality())
            throw MakeStringException(ECLWATCH_INVALID_INPUT, "File not specified.");
```
In a separate small loop
```
ForEachItemIn(idx, files)
{
    if (findPathSepChar(file))
        throw makeStringExceptionV(ECLWATCH_INVALID_INPUT, "Invalid file name %s", file);
}
```
That would make it so if any filenames were bad we wouldn't delete some but not all.

Reply:
I just changed. Please review.